<ui version="4.0" >
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Login window</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QComboBox" name="protocolComboBox" />
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QStackedWidget" name="stackedWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="page" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Login:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="wengoLoginLineEdit" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="wengoPasswordLineEdit" >
         <property name="echoMode" >
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Login / username:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3" >
        <widget class="QLineEdit" name="usernameLineEdit" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Password:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3" >
        <widget class="QLineEdit" name="passwordLineEdit" >
         <property name="echoMode" >
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Realm:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3" >
        <widget class="QLineEdit" name="realmLineEdit" />
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Displayname:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="3" >
        <widget class="QLineEdit" name="displaynameLineEdit" />
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>Server:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="3" >
        <widget class="QLineEdit" name="registerServerLineEdit" />
       </item>
       <item row="5" column="0" >
        <widget class="QLabel" name="label_8" >
         <property name="text" >
          <string>Proxy:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" >
        <widget class="QLineEdit" name="sipProxyLineEdit" />
       </item>
       <item row="5" column="2" >
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3" >
        <widget class="QLineEdit" name="sipProxyPortLineEdit" >
         <property name="text" >
          <string>5060</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QPushButton" name="loginButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Login</string>
     </property>
     <property name="autoDefault" >
      <bool>true</bool>
     </property>
     <property name="default" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="callButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Call...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
